### ISSUE TITLE
Version tags for docker images

### ISSUE THREAD

**Issue by RemDelaporteMathurin** [fusion-energy/fusion_neutronics_workflow#33]
Currently the only tag available for the docker image is latest.
Would it be possible to have tags for each release?

**Comment by shimwell** [comment]
Yes this can be done using tags. This action is nearly what you want. It just uploads to dockerrhub instead of ghcr

https://github.com/openmc-dev/openmc/blob/develop/.github/workflows/dockerhub-publish-release.yml#L13

**Comment by RemDelaporteMathurin** [comment]
I was just looking at this ;-)
